### PR TITLE
Doc update

### PR DESCRIPTION
**Pull request by regislon**
# Prework

- [x] I understand and agree to this repository's [code of conduct](https://github.com/ltelab/disdrodb/blob/main/CODE_OF_CONDUCT.md).
- [x] I understand and agree to this repository's [contributing guidelines](https://github.com/ltelab/disdrodb/blob/main/CONTRIBUTING.md).
- [x] I have already submitted an [issue](https://github.com/ltelab/disdrodb/issues) or [discussion thread](https://github.com/ltelab/disdrodb/discussions) to discuss my idea with the maintainers.

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [x] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and communicate accordingly:

**The PR fulfills these requirements:**

- [x] It's submitted to the branch named as follow : 
  - Add a reader: `reader-<institute>-<campaign>`
  - Fix a bug: `bugfix-<some_key>-<word>`
  - Improve the doc: `doc-<some_key>-<word>`
  - Add a new feature: `feature-<some_key>-<word>`
  - Refactor some code: `refactor-<some_key>-<word>`
  - Optimize some code: `optimize-<some_key>-<word>`
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] Don't forget to link PR to issue if you are solving one.
- [x] All tests are passing.
- [x] New/updated tests are included

If adding a **new feature**, the PR's description includes:

- [x] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

# Related GitHub issues and pull requests

- Ref: #129 #109 #19

# Summary

Update documentation

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -231,27 +231,74 @@ You should configure VS code as follow :
...
+
+	pytest disdrodb/test_readers
+
+To create a new test, simply add a small, single-station dataset and the associated files (issue, metadata), and expected data, in the following manner:
+
+| 📁 disdrodb/test_readers/
+| ├── 📁 test_ressources_raw_data
+|     	├── 📁 L0
+|     		├── 📁 readers
+|     	      ├── 📁 DISDRODB
```
Comment (ghiggi) [inline CONTRIBUTING.rst:278]: If we add the reader name to the metadata.yaml, I guess here:
- it's enough to create a directory for each reader_name
- and then for each reader directory we add directly a sample data and a dummy yaml no? 
- And then the test function creates the eventual required sub_directory stucture in a tmp directory no?

File: docs/source/data.rst [hunk 2]
```diff
@@ -0,0 +1,16 @@
+=========================
+Data
+=========================
+
+
+Access to Disdometer measurement datasets is available through a separate Github repository : `LTE-disdrodb-data <https://github.com/EPFL-ENAC/LTE-disdrodb-data>`__.
```
Comment (ghiggi) [inline docs/source/data.rst:6]:
```suggestion
Access to the disdrometer measurement datasets is available through a separate Github repository : `LTE-disdrodb-data <https://github.com/EPFL-ENAC/LTE-disdrodb-data>`__.
```

Comment (ghiggi) [reply docs/source/data.rst:6]: I gave you adming access. Move the repo to the ltelab, rename it disdrodb-data and then modify this accordingly?

File: docs/source/data.rst [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+
+This Github repository includes the following information:
```
Comment (ghiggi) [inline docs/source/data.rst:9]:
```suggestion
This GitHub repository includes the following information:
```

File: docs/source/data.rst [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+- Station measurements in the form of a JSON file with links to the downloadable data (not included in the repository due to Github's file size limitation).
```
Comment (ghiggi) [inline docs/source/data.rst:11]:
```suggestion
- Stations measurements in the form of a JSON file with links to the downloadable data (not included in the repository due to GitHub's file size limitation).
```

File: docs/source/data.rst [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+- Station metadatas in a yaml files.
```
Comment (ghiggi) [inline docs/source/data.rst:12]:
```suggestion
- Stations metadata in YAML files.
```

File: docs/source/data.rst [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+- Station issues in a CSV files.
```
Comment (ghiggi) [inline docs/source/data.rst:13]:
```suggestion
- Stations issues in YAML files.
```

Comment (ghiggi) [reply docs/source/data.rst:13]: This has not been yet implemented right?

File: docs/source/data.rst [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+- Python code to download the dataset using the URLs provided in the JSON file.
```
Comment (ghiggi) [inline docs/source/data.rst:14]:
```suggestion
- Python code to download the datasets using the URLs provided in the JSON files.
```

File: docs/source/install.rst [hunk 8]
```diff
@@ -5,15 +5,15 @@ Installation
 
 We define here two types of installation :
 
-- `Installation for users`_ : for users who want to process data.
+- `Installation for standard users`_ : for users who want to process data.
 
 - `Installation for contributors`_: for contributors who want to enrich the project (eg. add a new reader).
 
+We recommend users and contributors to use a `Virtual environment`_ to install Disdrodb.
```
Comment (ghiggi) [inline docs/source/install.rst:12]:
```suggestion
We recommend users and contributors to use a `Virtual environment`_ to install DISDRODB.
```

File: docs/source/install.rst [hunk 9]
```diff
@@ -60,7 +51,9 @@ According to the `contributors guidelines <contributors_guidelines>`__, you shou
 
 
 
-Install the python developing environment
+
+
+Install the disdrobd package dependencies
```
Comment (ghiggi) [inline docs/source/install.rst:56]:
```suggestion
Install the DISDRODB package dependencies
```

File: docs/source/install.rst [hunk 10]
```diff
@@ -205,3 +121,58 @@ indicates if the commit contained any formatting errors (that are automatically
 	To maintain consitency, we use Black version `22.8.0` (as defined into `.pre-commit-config.yaml`). Make sure to stick to version.  
 
 
+
+
+
+Virtual environment
+==================================
+
+While not mandatory, utilizing a virtual environment when installing disdroDB is Recommended. Using a virtual environment for installing packages provides isolation of dependencies, easier package management, easier maintenance, improved security, and improved development workflow.
```
Comment (ghiggi) [inline docs/source/install.rst:130]:
```suggestion
While not mandatory, utilizing a virtual environment when installing DISDRODB is recommended. Using a virtual environment for installing packages provides isolation of dependencies, easier package management, easier maintenance, improved security, and improved development workflow.
```

File: docs/source/install.rst [hunk 11]
```diff
@@ -205,3 +121,58 @@ indicates if the commit contained any formatting errors (that are automatically
...
+
+To set up a virtual environment, follow these steps :
+
+* **With venv :**  
+
+	* Windows: Create a virtual environment with venv:
+
+		.. code-block:: bash
+
+		   python -m venv venv
```
Comment (ghiggi) [inline docs/source/install.rst:142]: I would call the virtual environment also here disdrodb-dev !

File: docs/source/install.rst [hunk 12]
```diff
@@ -205,3 +121,58 @@ indicates if the commit contained any formatting errors (that are automatically
...
+			conda create --name disdrodb-dev python=3.9 --no-default-packages
+
+	* Activate the disdrodb conda environment:
+
+		.. code-block:: bash
+
+			conda activate disdrodb-dev
+
+		
+	* Manually add the path of your local copy of disdrodb to the ``PYTHONPATH`` environment variable.
```
Comment (ghiggi) [inline docs/source/install.rst:171]: This is not required now that `pip install -e .` works correctly.

**Review by ghiggi — COMMENTED** [review]
Some small comments ...

File: docs/source/data.rst (hunk 2 above)
Comment (charlottegiseleweil) [reply docs/source/data.rst:6]: @regislon ping to do this :)

**Review by charlottegiseleweil — APPROVED** [review]
install instructions so much clearer! fine with me to drop the conda install, pip install is the most common and works fine.

File: docs/source/install.rst (hunk 11 above)
Comment (regislon) [reply docs/source/install.rst:142]: ok done

File: docs/source/install.rst (hunk 12 above)
Comment (regislon) [reply docs/source/install.rst:171]: ok done

**Pull request merged**